### PR TITLE
Add HumanCount and template keys to print position and length with co…

### PR DESCRIPTION
**Pull request by dabreegster**
…mmas. #339

Motivation and trade-offs described in the issue. This was quite straightforward. Already using in https://github.com/dabreegster/rampfs/commit/7332180b1b5abea2e3741a20ae2684cbcc08a1d4, demo:
Screenshot from 2021-12-30 17-26-30

### CODE REVIEW

File: src/format.rs [hunk 1]
```diff
@@ -125,6 +129,23 @@ impl fmt::Display for BinaryBytes {
     }
 }
 
+impl fmt::Display for HumanCount {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        use fmt::Write;
+
+        let num = self.0.to_string();
```
Comment (dabreegster) [inline src/format.rs:136]: There's almost certainly a more efficient implementation. Let me know if I should work on this or benchmark it

Comment (djc) [reply src/format.rs:136]: Hmm, not sure there's an obviously more efficient implementation.

I do think the backwards counting loop variable is a little confusing, how about changing the loop to just use `for (i, c) in num.chars().enumerate()` and rename your current `i` to `len` (immutable), then maybe add a binding for `let pos = len - i;` or something?

**Review by djc — COMMENTED** [review]
This mostly looks good to me, one style nit and I'll be happy to merge it!

File: src/format.rs (hunk 1 above)
Comment (dabreegster) [reply src/format.rs:136]: Agreed, I think the new version is a bit simpler. Another idea is to iterate in reverse, but then that requires reversing the output string

**Pull request merged**